src/hoc/__test__/CreditsProvider.test.tsx

```
@@ -0,0 +1,173 @@
+        expect(screen.getByTestId('credits').textContent).toBe('0');
+        expect(screen.getByTestId('orders').textContent).toBe('0');
+    });
+
+    it('should not fetch credits when there is no user', () => {
+        // Mock no user state
+        vi.mocked(useAuth).mockReturnValue({
+            user: null,
+            isLoading: false,
+        } as any);
+
+        const axios = require('axios');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)